Podfile

```diff
@@ -12,7 +12,6 @@ target 'swiftgen' do
   common_pods
   pod 'Commander', '~> 0.8'
   pod 'StencilSwiftKit', '~> 2.3'
-  pod 'Yams', '~> 0.3'
```

Comment:
Why did Yams disappear from the Podfile? Is it because it's already a dependency of SwiftGenKit?

Reply:
Yes